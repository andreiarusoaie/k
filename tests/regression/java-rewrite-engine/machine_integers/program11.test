MyListItem(uvalueMInt(miMInt(sremMInt(mi(32, smaxMInt(32)), mi(32, -10))))),
MyListItem(overflowMInt(sremMInt(mi(32, smaxMInt(32)), mi(32, -10))))
