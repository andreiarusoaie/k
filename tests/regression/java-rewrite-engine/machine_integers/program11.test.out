<k>
    MyListItem ( 4080218932 ) , MyListItem ( false )
</k>
